modules/refresh.js

```diff
@@ -0,0 +1,8 @@
+const refreshBtn = document.querySelector('.refreshBtn');
+
+const refresh = () => {
+  window.location.reload();
+  return false;
+};
```

Comment:
- [ ] When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change. Currently, you are just trying to refresh the page and it doesn't work as expected. Let's fix that